plugins/quota-aware-fs/build.gradle

```diff
@@ -0,0 +1,6 @@
+esplugin {
+  description 'A bootstrap plugin that adds support for interfacing with filesystem that enforce user quotas.'
+  classname 'org.elasticsearch.fs.quotaaware.QuoteAwareFsPlugin'
```

Comment:
This shouldn't be necessary for bootstrap plugins, we should change the validation to only require it for isolated plugins.

Reply:
I've changed the validation and removed this line. I initially retained an empty `classname` in the properties file as it felt more consistent, but then realised that it would be more consistent to omit it since we omit `javaOpts` for bootstrap plugins. I'm partly being thrown off by all the "example plugin" preamble at the top of the file. Do we really want that there? We could use JSP comments to remove it from the build output?

Reply:
At one point the template file was also referenced from our docs as an example for plugin authors who built the plugin properties on their own. I don't have any preference on how it is done, but we should definitely continue to have such an example.